ABOUT.md

```diff
@@ -1,9 +1,72 @@
+
+### Shogun is open source
+
+* *free* and *community-based* [ML training and skills](#useshoguntolearn) not just for students
+
+You can try Shogun in our [cloud](https://cloud.shogun.ml/hub/home) (no installation required) or find out more about our [mission](#mission), [features](#features) and [history](#shogunhistory).
+
+
+
+
+
+
+## <a name="mission"></a> Mission
+
+Machine learning is powerful. Within the last two decades, machine learning has emerged from an eccentric academic field to transforming the way we live our everyday life. Its methods influence society and its tools have a billion-dollar impact on global economy. We want to make sure that the underlying machine learning algorithms are transparent and accessible. We believe that ML should not be a privilege of corporations or political power with the money to buy technical knowledge. 
+Anyone - scientists, data enthusiasts, non-profit organisations, hackers, citizens, journalists, students - should be able to use ML tools for free.
+
+The Shogun ML toolbox promotes this mission through its features (*accessible*, *open source* and *good*) and a focus on ML education and developing its contributors.
+
+## <a name="features"></a> Shogun features - *accessible*, *open source* and *good*
+
+### Accessible
+
+ * Shogun is **not limited to a single language**: you can use the toolbox through a unified interface from C++, Python, Octave, R, Java, Lua, C#, Ruby, etc. See [examples](http://shogun.ml/cookbook/latest/). Thus we are independent of trends in computing languages and importantly, it allows you to use Shogun as a vehicle to expose your algorithm to multiple communities. We use [SWIG](http://www.swig.org/) to enable *bidirectional* communication between C++ and all target languages.
+ * Shogun **runs natively under Linux/Unix, MacOS, and Windows**.
+ * Anyone can **try Shogun with minimal effort**: the Shogun [cloud](https://cloud.shogun.ml/hub/home) allows users to run a docker instance on Amazon’s AWS framework without any installation effort. All you need to do is to login via openID and connect to a jupyter notebook. Furthermore, we are continuously improving and simplifying the installation process for Shogun.
+ * Shogun offers **bindings to other sophisticated libraries including**: [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), [SVMLight](http://svmlight.joachims.org/), [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), [VowpalWabbit](http://www.hunch.net/~vw/), [Tapkee](http://tapkee.lisitsyn.me/), [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. See our [list of integrated external libraries](http://www.shogun-toolbox.org/page/about/contributions).
+
+### Free
+
+ * Shogun is **community based** and **non-commercial**.
```

Comment:
Maybe say why: So that companies can use the code without having to turn their code-base to GPL